src/tests/class_tests/openms/data/MetaboliteFeatureDeconvolution_neg_output.consensusXML

```diff
@@ -22,145 +22,120 @@
 			<UserParam type="intList" name="distinct_charges" value="[-3, -2, -1]"/>
 			<UserParam type="int" name="distinct_charges_size" value="3"/>
 			<UserParam type="int" name="pure_proton_features" value="3"/>
-		</consensusElement>
-		<consensusElement id="e_6880193271103366492" quality="0">
-			<centroid rt="100" mz="0" it="100"/>
+			<UserParam type="string" name="102" value="H-3O-1"/>
```

Comment:
If you would like to extract the information again it may be better to use a StringList with a set e.g. 
103:H-2Na1 - or something in this direction. What do you think?

Reply:
Mhh, possibly, gotta think about that if improvement for future, currently I just aim to make it the same as after FeatureLinking features with adducts.

Reply:
Currently I think this would provide no gains beside aesthetics: use cases require to read the elems (which contain further information), which provide the IDs to query the corresponding adduct  userParams.

Reply:
Ok - then leave it as it is for now.